### PR TITLE
Add instructions for setup

### PR DESCRIPTION
**Pull request by js-ts**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        http.HandleFunc("/version", serveVersion)
+
+        log.Println("Server is running on http://tmpget.bacalhau.org")
+        log.Fatal(http.ListenAndServe(":80", nil))
+}
+
+func serveInstallScript(w http.ResponseWriter, r *http.Request) {
+        // Serve the install.sh file from the disk.
+        http.ServeFile(w, r, "install.sh")
+}
```
Comment (simonwo) [inline server/server.go:24]: Can we just use a //go:embed here and that way we don't need to make sure the file is on the disk in the right place?

Comment (js-ts) [reply server/server.go:24]: Added

File: server/server.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+//go:embed install.sh
+var installScript embed.FS
+
+const latestVersion = "v1.0.3"
+const versionMessage = "A new version is available! \nUpdate by running the following command:\ncurl -sL https://get.bacalhau.org/install.sh | bash"
+
+type versionResponse struct {
+        Version string `json:"version"`
```
Comment (simonwo) [inline server/server.go:19]: See my comment in the other PR about using a `BuildVersionInfo` here.

**Review by simonwo — COMMENTED** [review]
Otherwise, yes this looks great.

File: server/server.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        Message string `json:"message"`
+}
+
+func main() {
+        http.HandleFunc("/install.sh", serveInstallScript)
+        http.HandleFunc("/version", serveVersion)
+
+        log.Println("Server is about to start on http://tmpget.bacalhau.org")
```
Comment (simonwo) [inline server/server.go:27]: This domain name is untrue now – let's not hardcode this.

**Review by simonwo — APPROVED** [review]

**Pull request merged**